labelbox/schema/project.py

```diff
@@ -46,6 +47,20 @@
 logger = logging.getLogger(__name__)
 
 
+def _validate_datetime(string_date: str) -> bool:
```

Comment:
Is there a utils etc package to move it to?

Reply:
Nope - right now we're using that only in project file scope, I would move it into utils once there's use case outside of that file